_site/assets/all-ie8.css

```diff
@@ -0,0 +1 @@
+@import url(node_modules/govuk-frontend/govuk/all-ie8.css);
```

Comment:
Do we still need to cater for IE8? Or is this brought in by default from the GOV.UK toolkit?

Reply:
This is by default in the design system - I doubt our audience will use ie8 but worth leaving in I think